include/odp/api/spec/ipsec.h

```diff
@@ -1148,14 +1157,29 @@ typedef struct odp_ipsec_in_param_t {
  * Outbound inline IPSEC operation parameters
  */
 typedef struct odp_ipsec_out_inline_param_t {
-	/** Packet output interface for inline output operation
+	/** Packet output interface for inline outbound operation
 	 *
 	 *  Outbound inline IPSEC operation uses this packet IO interface to
 	 *  output the packet after a successful IPSEC transformation. The pktio
 	 *  must have been configured to operate in inline IPSEC mode.
 	 */
 	odp_pktio_t pktio;
 
+	/** Packet output TM queue for inline outbound operation
+	 *
+	 *  If pktio is configured to use traffic manager (ODP_PKTOUT_MODE_TM),
+	 *  then application is required to provide 'tm_queue' to be used after
+	 *  successful IPSEC transformation. The pktio must have been configured to
+	 *  operate in inline IPSEC mode.
+	 *
+	 *  If application does not configure all pktio to use traffic manager, then
```

Comment:
This would not be needed since now the pktio indicates (through its output mode) wheter TM is to be used with inline IPsec or not.